lib/sticky-session.js

```diff
@@ -271,9 +270,10 @@ StickyAgent.prototype.listener = function(msg, socket) {
 
         searchPos = strData.indexOf(':', searchPos) + 1;
 
-        endPos = strData.search(/\r\n|\r|\n/, searchPos);
-        strData = strData.substr(searchPos, endPos).trim();
+        strData = strData.slice(searchPos);
 
+        var endPos = strData.search(/\r\n|\r|\n/);
```

Comment:
All variable declarations should always be at the top of the current scope and never be just throwed in.